src/sfapi_client/_sync/compute.py

```diff
@@ -134,7 +134,7 @@ def job(
     @check_auth
     def jobs(
         self,
-        jobids: Optional[int] = None,
+        jobids: Optional[Union[List[int], List[str]]] = None,
```

Comment:
Should this be `List[Union[int, str]]` as a user could mix `str` and `int` in the same list?